src/index.html

```diff
@@ -11,8 +14,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__block">
+        <img
+          src="./images/imac.jpeg"
+          alt="product card image"
+          class="product-card__image"
+        />
+      </div>
+
+      <div class="product-card__title">
```

Comment:
According to the BEM methodology, the element name should be formed as block name plus two underscores plus element name. In this case, it should be 'product-card__info' instead of 'product-card__title' to better reflect that it's a container for information about the product.